pipeline.py

```diff
@@ -72,31 +94,89 @@ class TextProcessingPipeline:
     """
 
     def __init__(self, corpus_manager: CorpusManager):
-        pass
+        self.corpus_manager = corpus_manager
 
     def run(self):
         """
         Runs pipeline process scenario
         """
-        pass
+        articles = self.corpus_manager.get_articles().values()
+        cleaned_tokens = []
+        single_tagged = []
+        multiple_tagged = []
+        for article in articles:
+            raw_text = article.get_raw_text()
+            tokens = self._process(raw_text)
+            for token in tokens:
+                cleaned_tokens.append(token.get_cleaned())
+                single_tagged.append(token.get_single_tagged())
+                multiple_tagged.append(token.get_multiple_tagged())
+            article.save_as(' '.join(cleaned_tokens), 'cleaned')
+            article.save_as(' '.join(single_tagged), 'single_tagged')
+            article.save_as(' '.join(multiple_tagged), 'multiple_tagged')
 
     def _process(self, raw_text: str):
         """
         Processes each token and creates MorphToken class instance
         """
-        pass
+        cleaned_text = ' '.join(re.findall(r'[а-яёА-ЯЁ]+', raw_text))
```

Comment:
I do not see a reason to make such a preprocessing. Mystem can accept "dirty" text (with all the punctuations and other staff) and then it will process it accordingly, you only need to properly parse its output report

Reply:
I cleared the text from punctuation, so without cleaning the program works for a very long time.